### PR TITLE
[efsr] 57481 - Spacing/consistency updates for overall FSR

### LINKED ISSUE #57481: [eFSR] Spacing consistency updates
## Issue Description
There is inconsistent spacing between titles and form elements throughout the form. Need to make them uniform.

---
## Tasks
- [x] Make spacing between title/label and form input fields uniform.

## Acceptance Criteria
- [x] Spacing will be consistent throughout the form. 
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

### PR DESCRIPTION
**Pull request by josephrlee**
## Summary

- Made inconsistent spacing between titles and form elements throughout the form uniform and cleaned up unnecessary styles.
- Fixed lining issues. 
- Added missing titles to Debt Selection, Vehicle Summary, Housing Expenses, and Employment Question pages. 
- Cleaned up the `edit` and `delete` buttons on the mini summary cards and made them closer to the design files.
- Created override `.schemaform-block-title` for `font-size` to be an `h3` and made all titles consistent.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#57481

## Testing done

- Viewed in browser to make sure correct styles and content were present

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

| &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Before | After |
| ------- | ------ | ----- |
| Available Debts  | staging va gov_manage-va-debt_request-debt-help-form-5655_all-available-debts | localhost_3001_manage-va-debt_request-debt-help-form-5655_all-available-debts |
| Work History | staging va gov_manage-va-debt_request-debt-help-form-5655_employment-question | localhost_3001_manage-va-debt_request-debt-help-form-5655_employment-question |
| Vehicle Summary | staging va gov_manage-va-debt_request-debt-help-form-5655_vehicles-summary | localhost_3001_manage-va-debt_request-debt-help-form-5655_vehicles-summary |

## What areas of the site does it impact?

Impacts the majority of the eFSR form. Updated lots of components and pages to make them consistent throughout. 

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.go header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by josephrlee** [comment]
This PR is ready for review! I didn't bother with documenting every single screenshot, just one screenshot to show each type of change was done, didn't want to spam y'all with like 50 screenshots. 

@Dr-Pongo, @OptionSelect, @amponce

File: src/applications/financial-status-report/components/SpousePayrollDeductionInputList.jsx [hunk 1]
```diff
@@ -99,28 +99,29 @@ const SpousePayrollDeductionInputList = props => {
...
-            currency
-          />
-        </div>
-      ))}
+      <fieldset className="vads-u-margin-y--2">
+        <legend className="schemaform-block-title">
+          Your spouse’s job at {employerName}
+        </legend>
+        <p>
+          How much does your spouse pay monthly for each of your payroll
```
Comment (amponce) [inline src/applications/financial-status-report/components/SpousePayrollDeductionInputList.jsx:107]: Change `your` to `their`

File: src/applications/financial-status-report/pages/income/employmentEnhanced/grossMonthlyIncome.js [hunk 2]
```diff
@@ -3,11 +3,8 @@ import GrossMonthlyIncomeInput from '../../../components/GrossMonthlyIncomeInput
 
 export const uiSchema = {
   grossMonthlyIncome: {
-    'ui:title': (
-      <span className="vads-u-font-size--h4 vads-u-font-family--sans">
-        What’s your gross <strong>monthly</strong> income at this job?
-      </span>
-    ),
+    'ui:title':
+      'What’s your gross <strong>monthly</strong> income at this job?',
```
Comment (amponce) [inline src/applications/financial-status-report/pages/income/employmentEnhanced/grossMonthlyIncome.js:7]: Follow up on wether we need ()

File: src/applications/financial-status-report/pages/income/employmentEnhanced/spouseGrossMonthlyIncome.js [hunk 3]
```diff
@@ -3,11 +3,8 @@ import SpouseGrossMonthlyIncomeInput from '../../../components/SpouseGrossMonthl
 
 export const uiSchema = {
   grossMonthlyIncome: {
-    'ui:title': (
-      <span className="vads-u-font-size--h4 vads-u-font-family--sans">
-        What’s your spouse’s gross <strong>monthly</strong> income at this job?
-      </span>
-    ),
+    'ui:title':
+      'What’s your spouse’s gross <strong>monthly</strong> income at this job?',
```
Comment (amponce) [inline src/applications/financial-status-report/pages/income/employmentEnhanced/spouseGrossMonthlyIncome.js:7]: Follow up on rendering strong tags

File: src/applications/financial-status-report/sass/financial-status-report.scss [hunk 4]
```diff
@@ -216,9 +219,10 @@
   }
 }
 
-.repayments {
+.repayments, .utility-bills {
   #root__title {
     height: auto;
+    font-size: $h3-font-size;
```
Comment (amponce) [inline src/applications/financial-status-report/sass/financial-status-report.scss:225]: revist

**Review by amponce — APPROVED** [review]
Great work! the changes look 🔥 🚀

**Pull request merged**